docs/source/tutorials/multifidelity/mf_pasha.rst

```diff
@@ -0,0 +1,89 @@
+PASHA: Efficient HPO and NAS with Progressive Resource Allocation
```

Comment:
The multifidelity tutorial is explaining basic methods and puts them in relation. It is not for advanced extensions like PASHA.

If you like, you can insert a paragraph and a link to the other tutorial somewhere, but nothing more. There are no independent sections for MOBSTER or Hyper-Tune either.

Reply:
Thanks, that makes sense - I've now added a paragraph about PASHA into the page about ASHA (there is now a link to the PASHA tutorial too).

Reply:
This file needs to move to tutorials/pasha/pasha.rst. And the img files have to move as well.

Reply:
I've now moved these files into a new pasha folder